.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -383,14 +401,6 @@ jobs:
           image-tag: ${{ steps.downgrade-branch.outputs.sha }}
           chart-dir: ./untrusted/cilium-downgrade/install/kubernetes/cilium
 
-      - name: Wait for images to be available (newest)
-        timeout-minutes: 10
-        shell: bash
-        run: |
-          for image in cilium-ci operator-generic-ci clustermesh-apiserver-ci ; do
-            until docker manifest inspect quay.io/${{ env.QUAY_ORGANIZATION_DEV }}/$image:${{ steps.newest-vars.outputs.sha }} &> /dev/null; do sleep 45s; done
-          done
-
       - name: Wait for images to be available (downgrade)
```

Comment:
Nit. It may make sense using the action here as well for consistency, but that will require https://github.com/cilium/cilium/pull/37901 to be merged first.